pkg/action/list.go

```diff
@@ -17,15 +17,19 @@ limitations under the License.
 package action
 
 import (
+	"os"
 	"path"
 	"regexp"
 
 	"k8s.io/apimachinery/pkg/labels"
 
+	"helm.sh/helm/v3/pkg/cli"
 	"helm.sh/helm/v3/pkg/release"
 	"helm.sh/helm/v3/pkg/releaseutil"
 )
 
+var settings = cli.New()
```

Comment:
I think this line can be removed in favor of:

`if err := l.cfg.Init(l.cfg.RESTClientGetter, "", os.Getenv("HELM_DRIVER"), debug); err != nil {`